oktapush_test.go

```diff
@@ -0,0 +1,11 @@
+package main 
+
+import (
+    
+    "testing"
+)
+
+func TestCredential (t *testing.T) {
+    oktaPush()
```

Comment:
It's not about changing `main` to `oktaPush`. You need `main` or else the program won't run. You need a function, and understand what you are trying to test. Since `oktalib` is already written. What you need to test is your writing stuff to a file function.  So you should have something like this:

```
func main(){
    // blablabla
    err := writeCredentialsToFile(writeCredentialsToFileInput{
                location: 
                towrite: 
     })
)

type writeCredentialsToFileInput struct {
     location string  // the location to write the content to
     towrite []byte   // the content to write
}
func writeCredentialsToFile(i writeCredentialsToFileInput) error {
       // the functionality where you write it to the file
}
```

Then in the `TestCredentials` test you test the function `WriteCredentialsToFile` function.